### PR TITLE
Fixed marshmallow error reporting

### PR DESCRIPTION
**Pull request by riklaunim**
This change is [Reviewable](https://reviewable.io/reviews/socialwifi/flask-jsonapi/44)

### CODE REVIEW

File: flask_jsonapi/resources.py [hunk 1]
```diff
@@ -68,7 +68,13 @@ def get(self, *args, **kwargs):
...
             data, errors = self.schema(include_data=include_fields, only=sparse_fields).dump(resource)
         except marshmallow.ValidationError as e:
-            return response.JsonApiErrorResponse.from_marshmallow_errors(e.messages)
+            raise exceptions.JsonApiException(detail='marshmallow.ValidationError', source=e.messages)
+        except (AttributeError, KeyError, ValueError) as e:
+            logger.error(
+                'Error Processing Request',
+                extra={'status_code': http.HTTPStatus.BAD_REQUEST, 'request': request, 'exception': e}
+            )
+            raise exceptions.JsonApiException(detail=str(e), source={'component': 'schema'})
```
Comment (Behoston) [inline flask_jsonapi/resources.py:77]: I'm not sure if we should return whole error as detail

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='flask-jsonapi',
-    version='0.10.1.dev0',
```
Comment (Behoston) [inline setup.py]: Leave 
`     version='0.10.1.dev0',`
releasing tool can handle versioning.

**Review by Behoston — CHANGES_REQUESTED** [review]

File: flask_jsonapi/resources.py (hunk 1 above)
Comment (riklaunim) [reply flask_jsonapi/resources.py:77]: it happens during development and there it's very helpful.

File: setup.py (hunk 2 above)
Comment (riklaunim) [reply setup.py]: ok

**Comment by jgoclawski** [comment]
LGTM :+1:

**Review by jgoclawski — APPROVED** [review]

**Pull request merged**